source/includes/harvest/_job_posts.md

```diff
@@ -250,3 +250,98 @@ questions | If present, will return an array of questions associated with this j
 
 <br>
 [See noteworthy response attributes.](#the-job-post-object)
+
+## Retrieve multiple job posts for a job
+
+```shell
+curl 'https://harvest.greenhouse.io/v1/jobs/{id}/job_posts'
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+[
+  {
+    "id": 129547,
+    "live": true,
+    "title": "Test job one",
+    "location": {
+      "name": "Material Plane"
+    },
+    "internal": false,
+    "external": true,
+    "job_id": 146218,
+    "content": "job post content",
+    "updated_at": "2016-09-19T14:04:25.297Z",
+    "internal_content": "",
+    "created_at": "2015-11-22T05:49:35.145Z",
+    "questions": [
+      {
+        "required": true,
+        "private": false,
+        "label": "First Name",
+        "type": "short_text",
+        "values": []
+      },
+      {
+        "required": true,
+        "private": false,
+        "label": "Last Name",
+        "type": "short_text",
+        "values": []
+      }
+    ]
+  },
+  {
+    "id": 129548,
+    "live": true,
+    "title": "Test job two",
+    "location": {
+      "name": "Material Plane"
+    },
+    "internal": false,
+    "external": true,
+    "job_id": 146218,
+    "content": "job post content two",
+    "updated_at": "2016-09-19T14:04:25.297Z",
+    "internal_content": "",
+    "created_at": "2015-11-22T05:49:35.145Z",
+    "questions": [
+      {
+        "required": true,
+        "private": false,
+        "label": "First Name",
+        "type": "short_text",
+        "values": []
+      },
+      {
+        "required": true,
+        "private": false,
+        "label": "Last Name",
+        "type": "short_text",
+        "values": []
+      }
+    ]
+  }
+]
+```
+
+We recently updated our job posts modal to allow a single job to have multiple job posts.
+
+Retrieve all the coresponding job posts for a given Job ID.
+
+
+### HTTP Request
+
+`GET https://harvest.greenhouse.io/v1/jobs/{id}/job_posts`
```

Comment:
Do you think it'd make sense to rename the old endpoint now that there is a concept of multiple job posts for a single job?  

`Retrieve job post for job`

What does the older endpoint return if there are multiple job posts?  The "main" job post?  Or just one at random?  Could we call it something like 

`Retrieve main job post for job`
`Retrieve first job post for job`